lib/perl/Genome/Model/Tools/Analysis/LaneQc/CompareSnpsBuildLanes.pm

```diff
@@ -143,8 +144,8 @@ sub execute {
 samtools pileup -vc -f $reference_file $alignment_file | perl -pe '\@F = split /\\t/; \\\$_=q{} unless(\\\$F[7] > 2);' > $dir/$lane_name.var
 gmt analysis lane-qc compare-snps --genotype-file $genotype_file --variant-file $dir/$lane_name.var > $dir/$lane_name.var.compare_snps
 COMMANDS
-                my $email_domain = Genome::Config::get('email_domain');
-                print `bsub -N -u $user\@$email_domain "$command"`;
+                my $email = $user->email;
+                print `bsub -N -u $email "$command"`;
```

Comment:
I like the new abstraction.  Question on the bsub usage... Should probably avoid using backticks and leverage param-binding API instead (like IPC::Open3).  I see double quotes around the `$command`, and that might be under sufficient control up to this point that we know no one is -- within the command -- using double quotes, but it is legal for [email addresses](https://en.wikipedia.org/wiki/E-mail_address#RFC_specification) to have double-quoting and spaces, too: `"My Name"@example.com`.  Unlikely here, but hey.  I figure if we can't sanitize the incoming string, we should use Perl's built-in sanitizers where possible.